src/migrations/1584619633448_txs.ts

```diff
@@ -154,29 +154,30 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
 
     // `coinbase` tx types
     coinbase_payload: 'bytea',
-
   });
 
+  pgm.createIndex('txs', 'contract_call_contract_id');
   pgm.createIndex('txs', 'index_block_hash');
-  pgm.createIndex('txs', 'parent_index_block_hash');
   pgm.createIndex('txs', 'microblock_hash');
-  pgm.createIndex('txs', 'type_id');
-  pgm.createIndex('txs', 'block_height');
-  pgm.createIndex('txs', 'status');
+  pgm.createIndex('txs', 'parent_index_block_hash');
+  pgm.createIndex('txs', 'sender_address');
+  pgm.createIndex('txs', 'smart_contract_contract_id');
   pgm.createIndex('txs', 'sponsor_address');
-  pgm.createIndex('txs', 'sponsored');
+  pgm.createIndex('txs', 'status');
   pgm.createIndex('txs', 'token_transfer_recipient_address');
-  pgm.createIndex('txs', 'contract_call_contract_id');
-  pgm.createIndex('txs', 'smart_contract_contract_id');
-  pgm.createIndex('txs', ['canonical', 'microblock_canonical']);
```

Comment:
Are you confident this is unnecessary? We have a lot of `WHERE canonical=true AND microblock_canonical=true` queries -- those never use this index?

Reply:
Yep, that's right. Since they're boolean properties, the filtering is done at the last step of the query plan (without using the index).

I tested this thoroughly with different queries and I never managed to get postgres to use this index.

Reply:
I'm other words, it's basically an index that has 4 possible values.

In a huge table it doesn't make a big difference.